### PR TITLE
updated parter and gcs doc

### PR DESCRIPTION
**Pull request by varunutreja**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/api/index.md [hunk 1]
```diff
@@ -41,34 +42,32 @@ Register to Globalization Content Service and setup a task polling framework/con
...
-1. **project**: The name of the project under which tasks are created
-1. **task:** Unique string that defines the task name. 
-### Curl Command
-
-```java
-curl -X GET "https://gcscore-dev-va7.stage.cloud.ADOBE.io /v1/projects/DemoProject/tasks/<task_name>/assets?orgId=<your_org>ADOBEOrg" -H "accept: application/json"
-```
-
+1. **projectId**: The unique string of the project under which tasks are created
+1. **taskId:** Unique string that defines the task name.
```
Comment (ravisingh326) [inline src/pages/api/index.md]: **taskId:** A unique id for the translation task

File: src/pages/api/index.md [hunk 2]
```diff
@@ -84,140 +83,101 @@ The following is the successful response with the 200 Response code.
...
-  "statusInfo": "OK",
-  "allowedMethods": [],
-  "stringHeaders": {},
-  "entityTag": null,
-  "links": [],
-  "headers": {}
+	"service": "GCS",
+	"response": [
+		{
+			"name": "<taskName>",
```
Comment (ravisingh326) [inline src/pages/api/index.md]: taskName should be replaced with assetName

File: src/pages/api/index.md [hunk 3]
```diff
@@ -84,140 +83,101 @@ The following is the successful response with the 200 Response code.
...
+- **sourceURL**: Publicly available URLs (S3 or Azure) for downloading the source asset, can be located in response object with parameter "sourceUrl". This is just for reference and not to be consumed for translation process.
+- **status**: 200 ok, indicating request has been executed successfully
+- **mimeType**: Defines the nature and format of the asset (media type)
 - **createdDate**: Date when the asset was created; represented in ISO 8601 spec; e.g. format 2021-08-05T18:35:58Z 
 - **assetMetadata**: Details of the asset name and asset path 
 - **assetLocale**: Information that helps define the source asset language. Details locale as fr-FR, status as Created and updatedDate value.
-- **entityType** is a java array list object
+- **assetUrls/url**: It is important to note here is that the actual asset URL to be used by the partner for translation is the "url" parameter in the response object under "assetUrls", which is in NORMALIZED state. The unique object key to locate this asset is below,
+```java
+<customer_org_id>/<ProjectId>/<TaskId>/normalised/<AssetName>.json/en-US/<AssetName>.json.xlf
```
Comment (ravisingh326) [inline src/pages/api/index.md]: normalized

File: src/pages/api/index.md [hunk 4]
```diff
@@ -441,377 +290,24 @@ See a sample XLIFF file.
...
 1. **orgId** The IMS organization id associated with your account. 
-1. **project**: The name of the project under which tasks are created
-1. **task:** Unique string that defines the task name. 
-### Curl Command
-
-```java
-curl -X PUT "https://gcscore-dev-va7.stage.cloud.ADOBE.io/v1/projects/DemoProject/tasks/TestTask806_2/assets/UserTest.html/locales/fr-FR/cancel?orgId=<your_org>ADOBEOrg" -H "accept: application/json"
-```
+1. **project**: The Id of the project under which tasks are created
+1. **task:** Unique string that defines the task Id.
```
Comment (ravisingh326) [inline src/pages/api/index.md]: **taskId:** A unique id for the translation task

**Review by ravisingh326 — CHANGES_REQUESTED** [review]

File: src/pages/api/index.md [hunk 5]
```diff
@@ -441,477 +320,64 @@ See a sample XLIFF file.
...
-
 The key parameters in response are:
 
-- **Project name**: DemoProject
-- **Asset name**: UserTest.html
-- **Task name**: TestTask806\_2
-- **Locale**: fr-FR [the locale of the asset to be cancelled]
+- **Project Id**: The Id of the project under which tasks are created
+- **Asset name**: An asset for which you want to cancel the translation workflow
+- **Task Id**: Unique string that defines the task Id
```
Comment (ravisingh326) [inline src/pages/api/index.md]: **taskId:** A unique id for the translation task.
Also, keep only relevant fields which are part of response.

**Review by ravisingh326 — APPROVED** [review]
Please update description for another taskId in doc.
Looks good to me.

File: src/pages/api/index.md (hunk 5 above)
Comment (varunutreja) [reply src/pages/api/index.md]: done

**Pull request merged**